.github/CODEOWNERS

```diff
@@ -102,6 +106,7 @@ extension/healthcheckextension/                                  @open-telemetry
 extension/healthcheckv2extension/                                @open-telemetry/collector-contrib-approvers @jpkrohling @mwear
 extension/httpforwarderextension/                                @open-telemetry/collector-contrib-approvers @atoulme
 extension/jaegerremotesampling/                                  @open-telemetry/collector-contrib-approvers @yurishkuro @frzifus
+extension/leaderelector/                                         @open-telemetry/collector-contrib-approvers @rakesh-garimella @k15r @skhalash @hisarbalik
```

Comment:
Do any of these codeowneres have any contributions to the OpenTelemetry project? We have a requirement that code owners must be OTel members. Some contributions required to apply for membership, see https://github.com/open-telemetry/community/issues/2541 as example

Reply:
ah okie then I will adjust the code owners files based on this requirement. I will need to apply for membership too as I am not an OTel member.